airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/TestDestination.java

```diff
@@ -515,7 +516,15 @@ private void pruneMutate(JsonNode json) {
       // likely did not exist in the original message. the most consistent thing to do is always remove
       // the null fields (this choice does decrease our ability to check that normalization creates
       // columns even if all the values in that column are null)
-      if (Sets.newHashSet("emitted_at", "ab_id", "normalized_at", "EMITTED_AT", "AB_ID", "NORMALIZED_AT").contains(key) || key.matches("^_.*_hashid$")
+      final HashSet<String> airbyteInternalFields = Sets.newHashSet(
```

Comment:
This could be a static final member

Reply:
Do we agree that this could will be simplified once we start prefixing columns with `airbyte`?

Reply:
yes, agreed.